<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Liikeradat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnTallenna">
      <property name="text">
       <string>Tallenna raportti</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabTiedot">
       <attribute name="title">
        <string>Potilaan tiedot</string>
       </attribute>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>190</y>
          <width>60</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>171</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Tutkimuksen päivämäärä</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabLonkka">
       <attribute name="title">
        <string>Lonkka</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabPolvi">
       <attribute name="title">
        <string>Polvi</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabVirheAs">
       <attribute name="title">
        <string>Virheasennot</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabNilkka">
       <attribute name="title">
        <string>Nilkka</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabAntrop">
       <attribute name="title">
        <string>Antrop. mitat</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabJalkat">
       <attribute name="title">
        <string>Jalkaterän asento</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabSpast">
       <attribute name="title">
        <string>Spastisiteetti</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabTasap">
       <attribute name="title">
        <string>Tasapaino</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnTallennaLom">
      <property name="text">
       <string>Tallenna lomake</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSulje">
      <property name="text">
       <string>Sulje</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
